tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -6110,6 +6142,10 @@ void ExecutionTest::RunComputeRawBufferLdStTest(D3D_SHADER_MODEL shaderModel, Ra
    CComPtr<IStream> pStream;
    ReadHlslDataIntoNewStream(L"ShaderOpArith.xml", &pStream);
 
+   // format shader source
+   char rawBufferTestShaderText[sizeof(RawBufferTestComputeShaderTemplate) + sizeof(RawBufferTestShaderDeclarations) + sizeof(RawBufferTestShaderBody)];
+   VERIFY_IS_TRUE(sprintf(rawBufferTestShaderText, RawBufferTestComputeShaderTemplate, RawBufferTestShaderDeclarations, RawBufferTestShaderBody) != -1);
```

Comment:
I'm also not a fan of C-style formatting functions, but it'd be easier to look the other way if you use the size-safe variants (snprintf, I think?)